app/views/hearings/_attendees.html.haml

```diff
@@ -1,28 +1,57 @@
 %div{ class: "govuk-!-margin-bottom-9" }
   %h2.govuk-heading-l
     = t('hearings.show.attendees.heading')
+
+  -# TODO: refactor: add method is_an_appeal? in the decorator
+  - is_an_appeal = hearing_details && hearing_details.hearing_type.description.include?('Appeal')
+
   .govuk-heading-s{ class: "govuk-!-margin-bottom-1" }
-    = t('hearings.show.attendees.defendants')
-  %p.govuk-body{ class: "govuk-!-margin-bottom-2", id: "defendants" }
-    - if hearing_details&.prosecution_cases&.any?
-      - cases = hearing_details.prosecution_cases
-      = safe_join(cases.map { |cs| cs.defendants.map { |df| "#{df.defendant_details.person_details.first_name} #{df.defendant_details.person_details.last_name}" } }, tag.br)
-    - else
-      = t('generic.not_available')
-  .govuk-heading-s{ class: "govuk-!-margin-bottom-1" }
-    = t('hearings.show.attendees.defence')
-  %p.govuk-body{ class: "govuk-!-margin-bottom-2", id: "defence" }
-    - if hearing_details&.defence_counsels&.any?
-      = safe_join([hearing.defence_counsels_list], tag.br)
+    - if is_an_appeal
```

Comment:
It's feeling like maybe rather than having multiple `if is_an_appeal` checks we should have 2 partials, and do a single `if` check and render the appropriate partial based on that?